source/NVDAObjects/window/winword.py

```diff
@@ -1382,6 +1141,9 @@ def _WaitForValueChangeForAction(self,action,fetcher,timeout=0.15):
 		return curVal
 
 	def script_toggleBold(self,gesture):
+		if not self.WinwordSelectionObject:
+			# No Object model, so just let the gesture through

```

Comment:
Perhaps these comments could be a little clearer? Correct me if I'm wrong, but once this is supported with UIA there will be an implementation here?